src/main/java/it/pagopa/pn/bff/service/InfoRecipientService.java

```diff
@@ -45,4 +49,33 @@ public Flux<BffPgGroup> getGroups(String xPagopaPnUid, String xPagopaPnCxId, Lis
 
         return pgGroups.map(GroupsMapper.modelMapper::mapGroups);
     }
-}
+
+    /**
+     * Get the list of PAs that use the PN
+     *
+     * @param xPagopaPnCxId   Public Administration id
+     * @param xPagopaPnCxType The type of the user
+     * @param paNameFilter    The prefix of the PA name
+     * @return The list of PAs
+     */
+    public Flux<PaSummary> getPaList(String xPagopaPnCxId, CxTypeAuthFleet xPagopaPnCxType, String paNameFilter) {
+        log.info("getPaList");
+
+        if (!List.of(CxTypeAuthFleet.PF, CxTypeAuthFleet.PG).contains(xPagopaPnCxType)
+                || xPagopaPnCxId.isEmpty()
+        ) {
+            throw pnBffExceptionUtility.wrapException(new WebClientResponseException(
```

Comment:
instead to do this, try to do something like this:

```
return Flux.error(new PnBffException(
        "Attachment idx not found",
        "The attachment idx is missed",
        HttpStatus.BAD_REQUEST.value(),
        ERROR_CODE_BFF_DOCUMENTIDNOTFOUND
));
```

Leave the BAD_REQUEST status, change the first and the second parameters (title and description of the error) and define a custtom code error (i.e.  ERROR_CODE_BFF_INVALIDBODY)